### PR TITLE
UI sync

### PR DESCRIPTION
**Pull request by harupy**
## Related Issues/PRs

 #xxx

## What changes are proposed in this pull request?

UI sync

## How is this patch tested?

- [ ] Existing unit/integration tests
- [ ] New unit/integration tests
- [x] Manual tests (describe details, including test results, below)

https://github.com/mlflow/mlflow/assets/17039389/0835885d-8fff-454c-93a7-99d34a6c4db3

## Does this PR change the documentation?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure the changed pages / sections render correctly in the documentation preview.

### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?

Components

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: mlflow/server/js/.gitignore
Comment (harupy) [inline mlflow/server/js/.gitignore:1]: Screenshot 2023-05-29 at 14 34 47

Comment (harupy) [reply mlflow/server/js/.gitignore:1]: Is this error ignorable? https://stackoverflow.com/a/50387233

Comment (harupy) [reply mlflow/server/js/.gitignore:1]: Reproduction:

https://github.com/mlflow/mlflow/assets/17039389/b0cefdde-91ba-492a-9dc0-557e0b81dd26

File: mlflow/server/js/src/common/components/CollapsibleSection.test.tsx [hunk 1]
```diff
@@ -1,3 +1,10 @@
+/**
+ * NOTE: this code file was automatically migrated to TypeScript using ts-migrate and
+ * may contain multiple `any` type annotations and `@ts-expect-error` directives.
+ * If possible, please improve types while making changes to this file. If the type
+ * annotations are already looking good, please remove this comment.
+ */
```
Comment (harupy) [inline mlflow/server/js/src/common/components/CollapsibleSection.test.tsx:6]: Looks like most of the changes is this.

File: mlflow/server/js/.gitignore
Comment (hubertzub-db) [reply mlflow/server/js/.gitignore:1]: Hey @harupy , it seems that the exception is caused by some DuBois internals that we have little control of. I think it's safe to ignore, here's the fix that filters it out from the devserver overlay
https://github.com/databricks/universe/pull/318001

**Comment by harupy** [comment]
https://github.com/mlflow/mlflow/assets/17039389/0835885d-8fff-454c-93a7-99d34a6c4db3

**Review by hubertzub-db — APPROVED** [review]
Left one comment about `UnityCatalogService`. I've read through all relevant changes (not the generated files) and all seems to be legitimate, I've done a quick sanity test with both development and bundled versions as well. As always, it'd be good to have some other eyes on it.
BTW The disparity of removed/added line count was concerning but it seems to be a result of design-system's change of bundling method.

File: mlflow/server/js/src/common/sdk/UnityCatalogService.ts [hunk 2]
```diff
@@ -1,3 +1,10 @@
+/**
+ * NOTE: this code file was automatically migrated to TypeScript using ts-migrate and
+ * may contain multiple `any` type annotations and `@ts-expect-error` directives.
+ * If possible, please improve types while making changes to this file. If the type
+ * annotations are already looking good, please remove this comment.
+ */
+
 import { getJson } from '../utils/FetchUtils';
 
 export class UnityCatalogService {
```
[the remark this answers is not in the document]
Comment (harupy) [reply mlflow/server/js/src/common/sdk/UnityCatalogService.ts:10]: We should. I remember the JS version of this file was exported before. I think we forgot to update the copybara config file to exclude it.

**Comment by harupy** [comment]
@hubertzub-db Thanks for the review!

**Pull request merged**